addons/stock/models/stock_rule.py

```diff
@@ -252,6 +252,10 @@ def _get_stock_move_values(self, product_id, product_qty, product_uom, location_
         :rtype: dictionary
         '''
         group_id = False
+        orderpoint_id = values.get('orderpoint_id') and values['orderpoint_id'].id
```

Comment:
``` python
# We only set `orderpoint_id` on the move if all the destination
# moves are in the same company as the rule in `self`. The idea is
# to avoid propagating orderpoints trough different companies.
```